src/Exception/InvalidArgument.php

```diff
@@ -10,4 +10,32 @@
  * @package Requests
  * @since   2.0.0
  */
-class InvalidArgument extends InvalidArgumentException {}
+final class InvalidArgument extends InvalidArgumentException {
+
+	/**
+	 * Create a new invalid argument exception with a standardized text.
+	 *
+	 * @param int    $position The argument position in the function signature. 1-based.
+	 * @param string $name     The argument name in the function signature.
+	 * @param string $expected The argument type expected as a string.
+	 * @param string $received The actual argument type received.
+	 *
+	 * @return \WpOrg\Requests\Exception\InvalidArgument
+	 */
+	public static function create($position, $name, $expected, $received) {
+		// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_debug_backtrace
+		$stack = debug_backtrace();
```

Comment:
To avoid wasting resources, the call should be done like this:
```suggestion
		$stack = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 2);
```